firmware/brake/kia_soul_ps/brake_control_module.ino

```diff
@@ -211,209 +260,377 @@ const double MIN_PACC = 2.3;              // minumum accumulator pressure to mai
 {
-	SMC->_sensor1_pin = sensor1_pin;
-	SMC->_sensor2_pin = sensor2_pin;
-	SMC->_control_pin = control_pin;
+	master_cylinder_solenoid._sensor1_pin = sensor1_pin;
+	master_cylinder_solenoid._sensor2_pin = sensor2_pin;
+	master_cylinder_solenoid._control_pin = control_pin;
 
-	pinMode( SMC->_control_pin, OUTPUT );
+	pinMode( master_cylinder_solenoid._control_pin, OUTPUT );
 
-	SMC_solenoids_open( SMC );
+	open_master_cylinder_solenoid( );
 }
 
-struct brakes_s
+//brake structure
+struct brake_data_s
 {
 	float _pressure_left = 0.0;            // last known right-side pressure
     float _pressure_right = 0.0;           // last known left-side pressure
-    byte _sensor_pin_left = 99;            // pin associated with left-side  pressure sensor
-    byte _sensor_pin_right = 99;           // pin associated with right-side pressure sensors
-    byte _solenoid_pin_left_a = 99;        // pin associated with MOSFET, associated with actuation solenoid
-    byte _solenoid_pin_right_a = 99;       // pin associated with MOSFET, associated with return solenoid
-    byte _solenoid_pin_left_r = 99;        // pin associated with MOSFET, associated with actuation solenoid
-    byte _solenoid_pin_right_r = 99;       // pin associated with MOSFET, associated with return solenoid
+    uint8_t _sensor_pin_left = 99;            // pin associated with left-side  pressure sensor
+    uint8_t _sensor_pin_right = 99;           // pin associated with right-side pressure sensors
+    uint8_t _solenoid_pin_left_a = 99;        // pin associated with MOSFET, associated with actuation solenoid
+    uint8_t _solenoid_pin_right_a = 99;       // pin associated with MOSFET, associated with return solenoid
+    uint8_t _solenoid_pin_left_r = 99;        // pin associated with MOSFET, associated with actuation solenoid
+    uint8_t _solenoid_pin_right_r = 99;       // pin associated with MOSFET, associated with return solenoid
     bool _increasing_pressure = false;     // used to track if pressure should be increasing
     bool _decreasing_pressure = false;     // used to track if pressure should be decreasing
     uint32_t _previous_millis = 0;    	   // will store last time solenoid was updated
-};
+} brakes;
 
-void brakes_depower_solenoids( brakes_s * const brakes )
+// *****************************************************
+// Function:    turn_off_all_brake_solenoids
+// 
+// Purpose:     Turn off all brake solenoids
+// 
+// Returns:     void
+// 
+// Parameters:  None
+// 
+// *****************************************************
+void turn_off_all_brake_solenoids( )
 {
-	analogWrite( brakes->_solenoid_pin_left_a, 0 );
-    analogWrite( brakes->_solenoid_pin_right_a, 0 );
-    analogWrite( brakes->_solenoid_pin_left_r, 0 );
-    analogWrite( brakes->_solenoid_pin_right_r, 0 );
+	analogWrite( brakes._solenoid_pin_left_a, 0 );
+    analogWrite( brakes._solenoid_pin_right_a, 0 );
+    analogWrite( brakes._solenoid_pin_left_r, 0 );
```

Comment:
Should have noticed before, what is a 'solenoid_pin_left_r' vs 'solenoid_pin_left_a'?  I think that I recall that the r is for 'release' and a is for 'actuator'.  Would make sense to make these clear.  I think the 'pin' part of it might not be useful so we could call it: solenoid_left_release and solenoid_left_actuate.  I would check with Evan first to find out what parts of the name are relevant.